src/main/java/org/activiti/cloud/query/QueryConfiguration.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2005-2019 Alfresco Software, Ltd. All rights reserved.
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+package org.activiti.cloud.query;
+
+import org.activiti.api.process.model.payloads.GetProcessDefinitionsPayload;
+import org.activiti.api.process.model.payloads.GetProcessInstancesPayload;
+import org.activiti.api.runtime.shared.security.SecurityManager;
+import org.activiti.core.common.spring.security.policies.ProcessSecurityPoliciesManager;
+import org.activiti.core.common.spring.security.policies.ProcessSecurityPoliciesManagerImpl;
+import org.activiti.core.common.spring.security.policies.SecurityPoliciesRestrictionApplier;
+import org.activiti.core.common.spring.security.policies.conf.SecurityPoliciesProperties;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import springfox.documentation.RequestHandler;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+
+import java.util.function.Predicate;
+
+@Configuration
+public class QueryConfiguration {
```

Comment:
I usually name it `...Config` and place it in configuration package but as no consensus on that I'm happy to leave it for now

Reply:
I prefer Configuration, but I moved it in a package